src/PlayFabBuddy.Infrastructure/Adapter/PlayFab/PlayerAccountAdapter.cs

```diff
@@ -11,6 +11,12 @@ public class PlayerAccountAdapter : IPlayerAccountAdapter
 {
     public async Task Delete(MasterPlayerAccountEntity account)
     {
+        var playedTitleList = await GetPlayedTitleList(account);
+        if (playedTitleList.MoreThanOne())
+        {
+            throw new Exception($"Master PlayerAccount ID \"{account.Id}\" has more than one Title");
```

Comment:
Why am I as a user getting this exception? I would be more verbose on the intend that folks using it understand why this is thrown.

Reply:
This will be printed as an error to console.

I was also considering introducing a custom exception class. What wwould you think about that idea?